src/app/libmain.cxx

```diff
@@ -171,6 +179,13 @@ option_context_new_with_commands (RpmOstreeCommandInvocation *invocation,
   else /* top level */
     g_string_append (summary, "Builtin Commands:");
 
+  int command_count = 0;
+  for (RpmOstreeCommand *command = commands; command->name != NULL; command++)
+    {
+      command_count++;
+    }
+
+  qsort (commands, command_count, sizeof (RpmOstreeCommand), &cmp_by_name);
```

Comment:
I think the `&` is unnecessary in `&cmp_by_name` here - just saying `cmp_by_name` is already a function pointer, we don't need a pointer-to-pointer.  I think the C language rules may coerce this back so it may work OK today, but let's just remove it.